### ISSUE TITLE
Add round stats to UI for QF

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-dapps-v2#3674]
Every QF round people are asking for stats during the round like:
- how much has been donated so far
- total number of donations
- total number of donor addresses

IT would be great to include in the frontend somehwere where you could check these stats in real time... basically something that queries our database.

Would love a suggestion on where to put it @mosaeedi 

Check out gitcoin's report card for an idea:

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL1AzQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--694a1c0570dfa9d66423c6f9fa1c8a6e7d1942f6/image.png" alt="image.png" />

https://reportcards.gitcoin.co/137/0x546b0bf47c5b5253ff83c8d614662adb7d3ba9b1

<img width="1437" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL1gzQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--16671a2372961f69cd0922bd829502413d6dc071/image.png" alt="image.png" />

<img width="678" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL1QzQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0c55b65b589ff56bcc2b6b0e30b783c311541b55/image.png" alt="image.png" />

**Comment by Tosinolawale** [comment]
![Projects list - Lauren and Marko](https://github.com/Giveth/giveth-dapps-v2/assets/35389241/de5b36b6-d36f-46ac-b19f-c08c4b7d7deb)
@laurenluz 
https://www.figma.com/file/C3HykACu0eNImhAqu0our8/Giveth-projects?type=design&node-id=5314%3A41254&mode=design&t=zsaRhwJ47mOpEk2p-1

**Comment by laurenluz** [comment]
left comments!

**Comment by Tosinolawale** [comment]
https://www.figma.com/file/C3HykACu0eNImhAqu0our8/Giveth-projects?type=design&node-id=5314%3A41254&mode=design&t=NM8Grnhc9H78txv1-1
Showing Stats for Active QF Rounds 
![Projects list - QF](https://github.com/Giveth/giveth-dapps-v2/assets/35389241/bd7dbbab-118e-4d63-a286-f1e5853f0151)

**Comment by laurenluz** [comment]
@jainkrati can you assign the appropriate devs for this? having the stats on the homepage like this will mean I have to bug @mohammadranjbarz & @CarlosQ96 a lot less for queries during the round :)

**Comment by jainkrati** [comment]
Looping in @mohammadranjbarz to see if BE needs to add these in the API response. If not, FE changes can be directly done by @MohammadPCh

**Comment by MohammadPCh** [comment]
@laurenluz @jainkrati, We also have another issue regarding the removal of the category selector component from the QF page(https://github.com/Giveth/giveth-dapps-v2/issues/2867). Do you think we can integrate these changes to improve the UX? Additionally, we discussed having an archive page for finished QF rounds, so it might be beneficial to include a link to that page if the round is finished.

**Comment by laurenluz** [comment]
that's a great idea @MohammadPCh - let's move ahead with removing category selectors for catergoies that are empty...

And I'll ask @Tosinolawale if we can put a mockup or where the link to "archived rounds" should go (I'll make a new issue).

**Comment by MohammadPCh** [comment]
Apologies, that was my mistake. There are three issues regarding QF pages:

1. Removing empty categories (I mentioned it's possible but not worth the effort) #3673
2. Implementing a home page to display all archived rounds (requires backend support) #3748
3. Creating a page to showcase archived rounds (requires backend adjustments) #3250

and also we have another issue that we want to show stats for ended round #3674 that I think we can show it on item 3.
wdyt @laurenluz ?

**Comment by laurenluz** [comment]
Ooh think you linked the wrong issue for categories.

1. Removing empty categories... ok agreed, let's just hide the categories then from the QF page
1. (b) #3673 - this one is ok right?

2 & 3 - ok I think these are great issues, but also if it's going to be complex w/ the backend RN, I think we can just complete adding the stats for the current round to the current (active) QF round page. I think for the post-round stats on the archived pages, we need to consider which fields we want to include in the design anyway.

**Comment by MohammadPCh** [comment]
@laurenluz Yes, #3673 looks good! 😄
Regarding point 2, I believe it's a straightforward backend task.
For point 3, the only remaining items are the supporting banner, its overlay images (4 images + support image), and the slug. However, I'm unsure if these have been implemented yet.

**Comment by laurenluz** [comment]
@jainkrati - we are blocked by this issue: https://github.com/Giveth/giveth-dapps-v2/issues/3665

**Comment by laurenluz** [comment]
I think @mohammadranjbarz needs to make end points for the live round stats

**Comment by MohammadPCh** [comment]
@mohammadranjbarz I use this query:
```
export const FETCH_QF_ROUND_STATS = gql`
	query fetchQfRoundStats($slug: String!) {
		qfRoundStats(slug: $slug) {
			uniqueDonors
			allDonationsUsdValue
			matchingPool
		}
	}
`;
```
let me know if i should use another one.

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz I use this query:
> 
> ```
> export const FETCH_QF_ROUND_STATS = gql`
> 	query fetchQfRoundStats($slug: String!) {
> 		qfRoundStats(slug: $slug) {
> 			uniqueDonors
> 			allDonationsUsdValue
> 			matchingPool
> 		}
> 	}
> `;
> ```
> 
> let me know if i should use another one.

@MohammadPCh Yes it's ok, but if you want more data to be included in the response just say it

**Comment by maryjaf** [comment]
Thanks @MohammadPCh These data has been added in QF page for current round
https://staging.giveth.io/qf/all
I have a question, "DAI" is set based on which field of QF round ?

<img width="1440" alt="image" src="https://github.com/Giveth/giveth-dapps-v2/assets/111529185/71450c8a-ca7c-4f64-b201-fe3a4ef07244">

**Comment by laurenluz** [comment]
right now we set the matching pool amount in USD... and use this value in our backend to detemined & show the estimated matching amount.

For now, this number can simply be the amount in USD... and we should show it as $ (not DAI).

But a longer term solution (and I will create an issue for this) is to allow us to set the matching pool amount & matching pool currency... and then use THAT to calculated the estimated matching etc... and THAT should be the number shown here.

Will create an issue now... but in the meantime, just make this $ and fetch the number from the "matching pool" amount in the backend.

**Comment by maryjaf** [comment]
> right now we set the matching pool amount in USD... and use this value in our backend to detemined & show the estimated matching amount

So for closing this issue it is needed that DAI be changed to $

**Comment by laurenluz** [comment]
exactly

**Comment by maryjaf** [comment]
> So for closing this issue it is needed that DAI be changed to $

This problem has been resolved
